src/routes/save-metadata.ts

```diff
@@ -0,0 +1,14 @@
+import { saveMetadata } from '../dependencies';
+import { RouteOptions } from 'fastify';
+
+const createEndpoint = ({ saveMetadata }): RouteOptions => ({
+  method: 'POST',
+  url: '/metadata',
+  handler: async (_req, reply) => {
```

Comment:
```suggestion
  handler: async (req, reply) => {
```
I think in the "/health" endpoint `req` is `_req` but that was because it was unused (prefixing it with an `_` ignores the eslint warning). Since we're using `req` now we can drop the `_` prefix.